server/package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "_comment": "we don't have a src folder right now, and until we make that, these scripts won't work.",
     "start": "node ./src/bin/www",
-    "dev": "PORT=3001 dotenv nodemon ./src/bin/www --watch src --ignore src/logs"
+    "dev": "SET PORT=3001 dotenv nodemon ./src/bin/www --watch src --ignore src/logs"
```

Comment:
It looks like this changed from the original (https://github.com/GTBitsOfGood/bright-futures-academy/blob/f78870ebbeed8adff20fb1fbd846720439a64ecb/server/package.json#L8)

To revert this file do `git checkout -- server/package.json` from the main bright-futures-academy folder!

Reply:
Actually, it looks like your branch is a bit behind. Don't worry about it, We can show you how to resolve merge conflicts